comet/utils/mediafusion.py

```
@@ -4,6 +4,42 @@
+            passwords_list = [None] * len(urls)
+        elif isinstance(passwords, str):
+            passwords_list = [passwords] * len(urls)
+        elif isinstance(passwords, list):
+            passwords_list = []
+            for i in range(len(urls)):
+                if i < len(passwords):
+                    passwords_list.append(passwords[i])
+                else:
+                    passwords_list.append(None)
+
+    return list(zip(urls, passwords_list))
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)